specs/experimental/plasma.md

```diff
@@ -83,6 +83,17 @@ on the `commitment_type_byte` where [0, 128) are reserved for official implement
 | `commitment_type` | `commitment`                    |
 | ----------------- | ------------------------------- |
 | 0                 | `keccak256(tx_payload)`         |
+| 1                 | `da-service`                     |
+
+The `da-service` commitment is as follows: `da_layer_byte ++ len(payload) as uvarint ++ payload`. The DA layer byte
+must be initially restricted to the range `[0, 127)`. This specification will not apportion DA layer bytes, but
+different DA layers should coordinate to ensure that the DA layer bytes do not conflict. The payload length is a
+unsigned 64 bit number, as defined in [protobuf spec]. We include the payload length so the op-node is able to pull
+out single commitment from a list of commitments that are concatenated together. The payload is a bytestring which
```

Comment:
Why are there multiple concatenated commits in the first place?

Reply:
That doesn't exist yet, but if we wanted to submit multiple commitments in the same L1 transaction, we would be able to. We would have to define a new L1 txn format, so could remove this data length byte